winton_kafka_streams/processor/_stream_thread.py

```diff
@@ -194,7 +259,8 @@ def run(self):
                     continue
```

Comment:
And this is the WIP bit, necessary mitigation #2: add more than just one record to inbound queue before going into the processing loop.
This probably requires looping on self.consumer.poll(0.) until it stops giving us new messages without waiting or we've gotten enough.